client-sdk/ts-web/rt/playground/sample-run-network.sh

```diff
@@ -38,7 +40,55 @@ jq '
 ' "$FIXTURE_FILE" >"$FIXTURE_FILE.tmp"
 mv "$FIXTURE_FILE.tmp" "$FIXTURE_FILE"
 
+# Run test runner to generate genesis document.
 "$TEST_NET_RUNNER" \
     --fixture.file "$FIXTURE_FILE" \
-    --basedir /tmp/oasis-net-runner-sdk-rt \
-    --basedir.no_temp_dir
+    --basedir "$WORKDIR" \
+    --basedir.no_temp_dir &
+RUNNER_PID=$!
+
+# Below is a workaround for there being no way to change the default max tx size which
```

Comment:
Do we have an issue in oasis-core, for adding support to this vis a fixture?

Reply:
Good point, I'll open an issue. Core otherwise has the increased max tx size in master.

Reply:
Done, https://github.com/oasisprotocol/oasis-core/issues/5960.

Reply:
Thx!